src/ansys/mapdl/core/examples/downloads.py

```diff
@@ -83,14 +83,11 @@ def _get_file_url(filename, directory=None):
 
 
 def _check_url_exist(url):
-    if not _HAS_REQUESTS:
-        raise ModuleNotFoundError("Examples module requires request module")
-
-    response = requests.get(url)
+    response = requests.get(url, timeout=10)  # 10 seconds timeout
     if response.status_code == 200:
```

Comment:
```suggestion
    return response.status_code == 200
```
Why not just do this return if you are just checking whether this evaluation is ``True`` or ``False``? Sorry for the "poor" suggestion but I can't refer to the underlying lines easily